src/gestalts/types.ts

```diff
@@ -14,7 +14,7 @@ import type {
 } from '../claims/payloads/index.js';
 import type { ProviderPaginationToken } from '../identities/types.js';
 
-const gestaltActions = ['notify', 'scan', 'claim'] as const;
+const gestaltActions = ['notify', 'scan', 'claim', 'join'] as const;
 
```

Comment:
I'm calling the permission to join a network `join`. Do we want a more descriptive name for this @CMCDragonkai ?